python_src/pyproject.toml

```diff
@@ -3,14 +3,15 @@ name = "lamp_py"
 version = "0.1.0"
 description = "Lightweight Application for Monitoring Performance"
 authors = [
-    "MBTA CTD <developer@mbta.com>",
-    "Ryan Rymarczyk <rrymarczyk@mbta.com>",
-    "Mike Zappitello <mzappitello@mbta.com>",
+  "MBTA CTD <developer@mbta.com>",
+  "Ryan Rymarczyk <rrymarczyk@mbta.com>",
+  "Mike Zappitello <mzappitello@mbta.com>",
 ]
 
 [tool.poetry.scripts]
 ingestion = 'lamp_py.ingestion.pipeline:start'
 performance_manager = 'lamp_py.performance_manager.pipeline:start'
+seed_metadata = 'lamp_py.postgres.seed_metadata:run'
```

Comment:
add this script so you can run `poetry run seed_metadata --env <env path> --seed-file <path to seed file>`